app/controllers/billDiscountSuppliers.js

```diff
@@ -88,16 +93,24 @@ module.exports = {
             }
           })
           .catch((e) => {
-            console.log(e);
+            console.log("Error In fetching seller profile is>>>>", e);
           });
 
-        let supplier = await BillDiscountSuppliers.create({
-          invited_by: profile.uuid,
-          status: "pending",
-          ...item,
-        });
+        let supplier = null;
+        const [inviteExists, inviteStatus] = await checkExistingInvites(
+          profile.uuid,
+          item.profile_uuid
```

Comment:
@ShreyDhyani What happens when email/mobile is not onboarded in wiredup?

Reply:
So here we need to handle the case if the invited user is not on wired up?? but we still need to send an invite???

Reply:
@ShreyDhyani Yes